public/SHACL/V2/valid-phone-number-constraint.ttl

```diff
@@ -0,0 +1,50 @@
+#####################################################
+# Valid phone number constraint shape for V2 forms ##
+#####################################################
+
+form:ValidPhoneNumber
+  a sh:NodeShape ;
+  sh:targetClass form:ValidPhoneNumber ;
+  sh:property
+    [
+      sh:path form:grouping ;
+      sh:hasValue form:MatchEvery ;
+      sh:minCount 1 ;
+      sh:maxCount 1 ;
+      sh:message "form:Grouping moet de waarde form:MatchEvery zijn."@nl ;
+      sh:severity sh:Violation;
+    ],
+    [
+      sh:path sh:path ;
+      sh:minCount 1 ;
+      sh:message "sh:path moet verwijzen naar het subject waar het op toegepast wordt."@nl ;
+      sh:severity sh:Warning;
+    ],
+    [
+      sh:path form:defaultCountry ;
+      sh:datatype xsd:string ;
+      sh:minCount 1 ;
+      sh:maxCount 1 ;
+      sh:in (
+        "BE"
+        "NL"
```

Comment:
over here

Reply:
In who's mind is adding two country codes to a phonenumber a thing  XD

I'll add it

Reply:
yeah mistakes are often not intentionally